language/move-prover/tests/sources/global_vars.move

```diff
@@ -0,0 +1,83 @@
+module GlobalVars {
+
+    spec module {
+        global sum_of_T: u64;
+    }
+
+    struct T {
+      i: u64,
+    }
+    spec struct T {
+      invariant pack sum_of_T = sum_of_T + i;
+      invariant unpack sum_of_T = sum_of_T - i;
+      invariant update sum_of_T = sum_of_T - old(i) + i;
```

Comment:
We are now using the "lifetime" model.  What does the expression old(i) in the update expression mean in this model?